documentation/dynawo_documentation.sh

```diff
@@ -27,7 +28,8 @@ echo '%% Except where otherwise noted, content in this documentation is Copyrigh
 
 \begin{document}
 
-\title{\Dynawo Documentation \\ \small{v1.2.0}}
+\title{\huge{\Dynawo Documentation} \\ \LARGE{v1.2.0} \\
```

Comment:
Not related to this commit but it makes me think about it: I am going to modify the master version also (1.3.0 probably) to differentiate it from the release branches